.travis.yml

```diff
@@ -1,4 +1,7 @@
 sudo: required
+jdk:
+  - openjdk8
+  - openjdk11
```

Comment:
@kensipe, is there a way to configure `build.gradle` as well?

Reply:
There's two fields with the Java Plugin:
`sourceCompatibility` and `targetCompatibility` [from here](https://docs.gradle.org/current/userguide/java_plugin.html#other_convention_properties).

It should correspond to [this line](https://github.com/mesosphere/marathon/blob/43cee2540108b67f4cf63946271c90e09bea563e/build.sbt#L100) from Marathon's SBT build.

From our conversations so far, `sourceCompatibility` is 1.8 but `targetCompatibility` can be 1.11.